stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1576,7 +1578,12 @@ public class ChatClient internal constructor(
     }
 
     public companion object {
-        private const val VERSION_PREFIX = "stream-chat-android-"
+        @InternalStreamChatApi
+        @JvmStatic
+        public var VERSION_PREFIX: String by Delegates.vetoable(VersionPrefixHeader.CORE_ANDROID.prefix) { _, _, new ->
```

Comment:
This could just have the enum type instead of a `String`, would simplify a bunch of things

Reply:
Very good point - I'll update!